src/main/java/com/google/cloud/resourcemanager/ResourceManagerImpl.java

```diff
@@ -273,6 +273,141 @@ public Policy replacePolicy(final String projectId, final Policy newPolicy) {
     }
   }
 
+  @Override
+  public Lien createLien(final LienInfo lienInfo) {
+    final com.google.api.services.cloudresourcemanager.model.Lien lien = lienInfo.toPb();
+    try {
+      return Lien.fromPb(
+          this,
+          runWithRetries(
+              new Callable<com.google.api.services.cloudresourcemanager.model.Lien>() {
+                @Override
+                public com.google.api.services.cloudresourcemanager.model.Lien call() {
+                  return resourceManagerRpc.createLien(lien);
+                }
+              },
+              getOptions().getRetrySettings(),
+              EXCEPTION_HANDLER,
+              getOptions().getClock()));
+    } catch (RetryHelperException ex) {
+      throw ResourceManagerException.translateAndThrow(ex);
+    }
+  }
+
+  @Override
+  public void deleteLien(final String name) {
+    try {
+      runWithRetries(
+          new Callable<Void>() {
+            @Override
+            public Void call() {
+              resourceManagerRpc.deleteLien(name);
+              return null;
+            }
+          },
+          getOptions().getRetrySettings(),
+          EXCEPTION_HANDLER,
+          getOptions().getClock());
+    } catch (RetryHelperException ex) {
+      throw ResourceManagerException.translateAndThrow(ex);
+    }
+  }
+
+  @Override
+  public Lien getLien(final String name) {
+    try {
+      com.google.api.services.cloudresourcemanager.model.Lien answer =
+          runWithRetries(
+              new Callable<com.google.api.services.cloudresourcemanager.model.Lien>() {
+                @Override
+                public com.google.api.services.cloudresourcemanager.model.Lien call() {
+                  return resourceManagerRpc.getLien(name);
+                }
+              },
+              getOptions().getRetrySettings(),
+              EXCEPTION_HANDLER,
+              getOptions().getClock());
+      return answer == null ? null : Lien.fromPb(this, answer);
+    } catch (RetryHelperException ex) {
+      throw ResourceManagerException.translateAndThrow(ex);
+    }
+  }
+
+  private static class LienPageFetcher implements NextPageFetcher<Lien> {
```

Comment:
outer class?

Reply:
this class used only one place so i would like to kept it, as it is.